PageObjectPattern/src/test/java/ShoppingStepDifinitions/TestRunner.java

```diff
@@ -0,0 +1,14 @@
+package ShoppingStepDifinitions;
+
+import org.junit.runner.RunWith;
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+
+//To run Feature test suits with specified tags & save reports in different formats
+@RunWith(Cucumber.class)
+@CucumberOptions(features = "src\\test\\resources\\Features", glue = { "ShoppingStepDifinitions" }, plugin = { "pretty",
```

Comment:
Path to features can be replaced with "classpath:features". Also rename Features -> features

Reply:
done